standard_reports/item-usage/latest/convert_data_js/codegen.yaml

```diff
@@ -0,0 +1,24 @@
+overwrite: true
+# Server must be running and initialised
+schema: 'http://localhost:8000/graphql'
```

Comment:
decided to go with this, otherwise have to generate types from schema file, which also needs to be generated, and we need to know where server is for that, which makes file structure quite dependent, wanted to avoid dependency we had previously, with report cli/bash etc..